airbyte-cdk/java/airbyte-cdk/dependencies/src/main/kotlin/io/airbyte/commons/string/Strings.kt

```diff
@@ -6,18 +6,20 @@ package io.airbyte.commons.string
 import java.util.*
 import org.apache.commons.lang3.RandomStringUtils
 
-object Strings {
-    fun join(iterable: Iterable<Any>, separator: CharSequence): String {
-        return iterable.joinToString(separator) { it.toString() }
-    }
+class Strings {
```

Comment:
object Strings is the canonical way to do that. You should revert this change, and make the methods @JvmStatic